scripts/install-base-packages.sh

```diff
@@ -27,7 +27,11 @@ apt-get update
 apt-get -y upgrade
 
 # Example of installing a new package, without unnecessary packages:
-apt-get -y install --no-install-recommends git
+# apt-get -y install --no-install-recommends git
```

Comment:
You will need git installed if you're using `setuptools_scm` (which `setup.cfg` is still referencing).

Reply:
Yes. I just discovered that when the github tests failed on pushing the Docker image. I fixed that and the github tests passed. I then rebased everything and am hoping the github tests *still* pass.